### PR TITLE
[-] BO : Fixed checking if module is trusted PSCSX-6883

### PR DESCRIPTION
**Pull request by brouillon**
Make the check case insensitive

### CODE REVIEW

File: classes/module/Module.php [hunk 1]
```diff
@@ -1690,6 +1690,8 @@ public static function getModulesInstalled($position = 0)
 
         $context = Context::getContext();
 
+	$module_name = Tools::strtolower($module_name);
```
Comment (gRoussac) [inline classes/module/Module.php]: Hi,

You need to uses spaces, not tab.

Thank you.

**Comment by gRoussac** [comment]
@jnadaud do you validate this ? thank you

**Comment by jnadaud** [comment]
Hi, let me check any possible side effect.
Regards.

**Comment by gRoussac** [comment]
http://forge.prestashop.com/browse/PSCSX-6883

**Comment by jnadaud** [comment]
Hi, 

Please use stripos instead at ligne 1724, 1726, 1728 et 1734, it will prevent a crash of Tools::file_exists_no_cache(_PS_MODULE_DIR_.$module_name.'/'.$module_name.'.php') at line 1113

Best regards.

**Comment by gRoussac** [comment]
Hi,

isModuleTrusted() is calling checkModuleFromAddonsApi() which is calling getInstanceByName() wich has a Tools::file_exists_no_cache(_PS_MODULE_DIR_.$module_name.'/'.$module_name.'.php') whic can break in possibel cases. @jnadaud suggest to replace strpos by stripos in comparisons.

Regards

**Comment by gRoussac** [comment]
:no_mouth:

**Comment by brouillon** [comment]
OK i have replaced strtolower call by stripos

**Comment by gRoussac** [comment]
Thank you for your PR and contribution. @jnadaud  thanks for your help.

Reagrds

**Pull request merged**